### PR TITLE
Pt134004809 create webhook for collector

### PR DESCRIPTION
**Pull request by aaronbassett**
# Add ability to create Webhooks for collectors as well as surveys

https://www.pivotaltracker.com/story/show/134004809

---

## Checklist

- [x] Meets criteria
- [x] Is modular & tested
- [x] Readable code
- [x] No duplication
- [x] Clear commits
- ~Documentation updated~
- [x] Handled error cases
- [x] **Code always improving**

### CODE REVIEW

File: surveymonkey/tests/webhooks/test_create_webhooks.py [hunk 1]
```diff
@@ -17,17 +17,23 @@
...
+    (DISQUALIFIED, COLLECTOR, "create_for_collector", str(random.randint(1234, 567890))),
+    (UPDATED, COLLECTOR, "create_for_collector", str(random.randint(1234, 567890)))
 ]
 
 
-multiple_survey_list = [
-    (COMPLETED, SURVEY, create_random_number_of_object_ids()),
-    (DISQUALIFIED, SURVEY, create_random_number_of_object_ids()),
-    (UPDATED, SURVEY, create_random_number_of_object_ids())
+multiple_obejct_list = [
```
Comment (richardtbell) [inline surveymonkey/tests/webhooks/test_create_webhooks.py]: typo "obejct"

File: surveymonkey/tests/webhooks/test_create_webhooks.py [hunk 2]
```diff
@@ -39,8 +45,8 @@ def setup_class(self):
     def setup_method(self, method):
         pass
 
-    @pytest.mark.parametrize("event_type,object_type,object_ids", single_survey_list)
-    def test_webhook_create_responses_for_single_survey(self, event_type, object_type, object_ids):
+    @pytest.mark.parametrize("event_type,object_type,method,object_ids", single_object_list)
```
Comment (richardtbell) [inline surveymonkey/tests/webhooks/test_create_webhooks.py]: ```"event_type,object_type,method,object_ids"``` - should these have spaces between them?

**Comment by aaronbassett** [comment]
@richardtbell can you check you're happy with this now and merge please?

**Comment by gicoupar** [comment]
LGTM

**Pull request merged**